### ISSUE TITLE
Game is crashing every time I try to load what I previously saved 

### ISSUE THREAD

**Issue by AAW9014** [OpenRCT2/OpenRCT2#19090]
### Operating System

windows 11, 64-bit

### OpenRCT2 build

OpenRCT2, v0.4.2 (8ceea45) provided by GitHub

### Base game

RollerCoaster Tycoon 2, RollerCoaster Tycoon Classic

### Describe the issue

when I attempt to open a game I've saved, It does not open. I get the message that the game has crashed. 

-It seems like it happens when I select certain rollercoasters (or maybe all rollercoasters) from the object selection menu.

I really don't know the cause but I've spent a lot of time designing my parks layouts and scenery and then when I start adding coasters, it crashes.

Today I attempted to open one of my saved games that crashed and it opened surprisingly but trying to save my work, It crashed again. 

### Steps to reproduce

1. Open game
2. Go to load Game
3. Click the saved game I want to open
4. Game crashes

Additional Issue
1. Open Game
2. Load game
3. saved game opens
4. try to re-save 
5. game crashes

### Attachments

[51b6ad2f-bc0b-40cd-bcde-764206c66265.zip](https://github.com/OpenRCT2/OpenRCT2/files/10369285/51b6ad2f-bc0b-40cd-bcde-764206c66265.zip)
[51b6ad2f-bc0b-40cd-bcde-764206c66265(8ceea45_x86).zip](https://github.com/OpenRCT2/OpenRCT2/files/10369288/51b6ad2f-bc0b-40cd-bcde-764206c66265.8ceea45_x86.zip)
![rct error message pic](https://user-images.githubusercontent.com/122225889/211214781-adbe2ccb-e151-4c58-b8ef-79778f818687.jpg)

**Comment by Gymnasiast** [comment]
You are on an old version. Please test this with the latest `develop` version first.

**Comment by AAW9014** [comment]
> You are on an old version. Please test this with the latest `develop` version first.

hey, thanks for responding.  I actually had the newer version first and was getting the same issues. I was reading some solutions online and it said using an older version might be more stable, so that's what I did. but I will will re-download it again and try.

**Comment by AAW9014** [comment]
Update.
I'm glad I sent this issue because when I went to re-download the latest version, I think i realized what the glitch was...  I didn't download the windows launcher version. [newbie] I looked at a youtube video and they mentioned that part and i was able to download it differently this time. So far im actually able to open all my old saved games that were crashing before. That's a big relief. If anything else seems buggy I will update you. Big Thanks for now
